lps22hb/lps22hb_nano_33_ble.go

```diff
@@ -0,0 +1,28 @@
+// +build nano_33_ble
+
+package lps22hb
+
+import (
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers"
+)
+
+// New creates a new LPS22HB connection. The I2C bus must already be
+// configured.
+//
+// This function only creates the Device object, it does not touch the device.
+func New(bus drivers.I2C) Device {
+	// turn on internal power pin (machine.P0_22) and I2C1 pullups power pin (machine.P1_00)
+	// and wait a moment.
+	ENV := machine.P0_22
+	ENV.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	ENV.High()
+	R := machine.P1_00
+	R.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	R.High()
+	time.Sleep(time.Millisecond * 10)
```

Comment:
I'm slightly uncomfortable with adding board specific code in a driver. This feels like a layering violation to me. It seems to me this is something an application should do (or some sort of board package, which we currently don't provide for TinyGo). @deadprogram any opinion here?

Reply:
I have no opinions really, but my only concern is that if you implement this in board package, there would have to have means to turn on/off the internal power at will, since Nano 33 BLE is very popular for IoT/TinyML projects and may need to save battery power.

Reply:
Hmm, looking at this specifically, along with the other drivers that are onboard the Nano 33 BLE, seems like we would like a balance between convenience and architectural integrity.

As far as the `New()` function, we'd probably prefer to keep that board agnostic. It would be better to use the `Configure()` function for this, since that would facilitate testing similarly to how other I2C devices are being tested in this package.

Having "magic pins" like `machine.P0_22` and `machine.P1_00` seems non-intuitive. I'd rather see something added to the main board definition such as: 

```go
const (
	I2C1_PullupPower Pin = P1_00
)
```

At least if a special `Configure()` protected by a build tag really does have to be created, it is easier to see what it is doing and why.

What do you think?